src/environments/environment.ts

```diff
@@ -0,0 +1,3 @@
+export const environment = {
+  gene_list_url: 'https://morphic-genelist-app.shinyapps.io/shiny_genesetexplore/'

```

Comment:
I approve now, but we could discuss whether the entire url belongs here, or alternatively, just the server part `https://morphic-genelist-app.shinyapps.io`, with possibly the resource path using a different config param.

Reply:
Thanks, sure we can discuss 👍